spec/spec_helper.rb

```diff
@@ -1,30 +1,33 @@
 if ENV['COVERAGE']
   require 'simplecov'
   SimpleCov.start do
-    add_filter "spec/"
+    add_filter 'spec/'
   end
 end
 
 require 'support/fixture_file'
 
 RSpec.configure do |config|
-  config.example_status_persistence_file_path = "spec/examples.txt"
+  config.example_status_persistence_file_path = 'spec/examples.txt'
   config.include FixtureFile
 
-  config.before(:suite) do |example|
-    return unless adapter = ENV["SAXERATOR_ADAPTER"]
+  adapter = ENV['SAXERATOR_ADAPTER']
+
+  config.before(:suite) do |_|
+    unless adapter
+      puts 'SAXERATOR_ADAPTER weren\'t defined'
```

Comment:
I don't think this should exit here, it should just skip this block. I think the `return` here probably should have been a `next`